src/extensions/report-portal-history.js

```diff
@@ -28,7 +28,7 @@ function getSymbols({ target, extension, launches }) {
   const symbols = [];
   for (let i = 0; i < launches.length; i++) {
     const launch = launches[i];
-    const launch_url = `${extension.inputs.url}/ui/#${extension.inputs.project}/launches/all/${launch.uuid}`;
+    const launch_url = `${extension.inputs.url}/ui/#${extension.inputs.project}/launches/all/${launch.launchId}`;
```

Comment:
If `launch.uuid` doesn't work it could be a bug from ReportPortal end.
Can you make this configurable via `inputs`.

```js
{
  "name": "report-portal-history",
  "inputs": {
    "link_history_via": "launchId"
  }
}
```

The default value of `link_history_via` should be `uuid`.

Reply:
Added a new commit as requested. Could you check?